app/src/components/ReviewList/index.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import { Stack } from "@mui/system";
+
+const ReviewList = ({ reviews, address }) => {
+  return (
+    <Box>
+      <Typography variant="h5" mb={2}>
+        What do students have to say about {address}?
+      </Typography>
+      {reviews ? (
+        <Stack spacing={3}>
+          {reviews.map((review) => (
+            <Box
+              key={review.review_id}
+              p={2}
+              border="1px solid #Dde6ea"
+              borderRadius={2}
+            >
+              <Typography>{review.comment}</Typography>
```

Comment:
This might be an issue when we are sorting, but I feel like we should display the review_helpfulness somehow. Otherwise, the user won't know what they are sorting by.

Reply:
oh yeah i agree, include review_helpfulness in the future. ill send a wireframe in teh gc. otherwise lgtm